ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -401,6 +402,12 @@ export default defineComponent({
       return hasNoOwners && index === 0
     }
 
+    const getGroupNumberById = (groupId: number): number => {
+      const activeOwnerGroups = getTransferOrRegistrationHomeOwnerGroups()
+        .filter(group => group.action !== ActionTypes.REMOVED)
+      return (activeOwnerGroups.findIndex(group => group.groupId === groupId)) + 1
+    }
+
```

Comment:
could this go into utils?

Reply:
It's not really generic functionality. 
It's pretty specific to finding groups by ID, amongst a collection of active groups. 

I suppose it could be refactored down to take various arguments to replace the data set we're searching and the property we're searching for but i don't see a lot of value. 

If you feel strongly about this though, can do!

Reply:
Nope, you're correct! Thanks for clarifying, was mostly just curious :)

Reply:
No prob! It's a totally valid consideration, something we should always be thinking about. 

I'm guilty of also forgetting to search through all the utils we have. Often we will create stuff locally to components or views when there is functionality existing external that could do the trick!